quest.py

```diff
@@ -0,0 +1,51 @@
+"""
```

Comment:
This needs to be moved into the `quest` folder as well. We usually want to call it something `CLI` for command line interface.